### DISCUSSION TITLE
Hide popup links

### DISCUSSION THREAD

**Discussion by cabmaxi** [elementor/elementor#21933] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It bothers me a little to see the link that is automatically generated when you mouse over a button or link that opens a popup.

### Describe the solution you'd like

It would be great to have the option to hide that link or have it hidden by default or not appear at all.

### Describe alternatives you've considered

I haven't really considered any temporary solutions because I believe there is no option to achieve what I am looking for.

**Comment by imihaiart** [comment]
Later edit: I have found a **workaround**.

### Steps for workaround:

1. In the Popup Settings go to **Advanced(tab) -> Advanced(section)** 
2. At option **Open By Selector** set an element ID which you'll use for the trigger element (Button or whatever you want as trigger).
Tip: Use the ID with the # (example: #popup-trigger)
3. Go to the widget(example: Button) that you'll use as trigger element for your popup and set the ID in the **Advanced** tab without the "#" (example: popup-trigger)

### Warning!
**Don't use "#"** as link if you want to have the pointer cursor on your trigger element. Using "#" as link will get you to the top of the page when you'll click the element to open the Popup.

### If you want the cursor to be set to "pointer" (for better UX):
1. Elementor (non pro):
- Add a custom CSS for your trigger element ID in the Wordpress Customized or any other methods/plugins to add custom CSS:
```
#popup-trigger {
cursor:pointer;
}
```
2. Elementor Pro
- Go to the element that you'll use as trigger element for your popup and put the custom css in the **Advanced -> Custom CSS**
```
selector {
    cursor:pointer;
}
```

I hope this will help someone until the devs will include this option.

> +1 
> 
> I have the same problem. It's not professional to show that link.
> Don't think this would be so hard working to be implemented. 
> 
> C'moon devs! I know you can do it!
>

**Reply by cabmaxi** [reply]
Thanks mihaiart, I will definitely give it a try!
